packages/livechat/src/lib/hooks.ts

```diff
@@ -247,6 +247,27 @@ const api = {
 		});
 	},
 
+	transferChat: async (department: string) => {
+		const {
+			config: { departments = [] },
+			room,
+		} = store.state;
+
+		const dep = departments.find((dep) => dep._id === department || dep.name === department)?._id || '';
+
+		if (!dep) {
+			throw new Error(
+				'The selected department is invalid. Check departments configuration to ensure the department exists, is enabled and has at least 1 agent',
+			);
+		}
+		if (!room) {
+			throw new Error("Conversation has not been started yet, can't transfer");
```

Comment:
Other thing: if these errors are going to be seen by visitors, is there a way of translating them?

If these are intended _only_ for developers and won't appear on UI, then pls ignore this and the above comment